.changeset/mean-foxes-hope.md

```diff
@@ -0,0 +1,10 @@
+---
+'skeleton': patch
+---
+
+Update `@shopify/cli` dependency in `package.json`:
+
+```diff
+-   "@shopify/cli": "3.51.0",
++   "@shopify/cli": "3.52.0",
+```
```

Comment:
@juanpprieto do we need this kind of changeset? Or should we just rely on `h2 upgrade`?

Reply:
I think we do need this changeset so that it would trigger an overall update?

Should add `'@shopify/cli-hydrogen': patch` as well so that newly generated skeleton template will have the updated cli package version